### DISCUSSION TITLE
PIC 16F1947

### DISCUSSION THREAD

**Discussion by IronMan1314** [lcgamboa/picsimlab#72] [General]
Can the PIC 16F1947 be added for simulation?
Thanks

**Comment by lcgamboa** [comment]
Hi @IronMan1314 ,

I will try to add the PIC16F1947 until next weekend and give you feedback here.

**Comment by lcgamboa** [comment]
Hi @IronMan1314 ,

the PIC16F1947 as added to PICSimLab in commit https://github.com/lcgamboa/picsimlab/commit/414b2c379c767ef4c95effb815fc02042092b9de . You can try it on the [lastest code build](https://github.com/lcgamboa/picsimlab/releases/tag/latestbuild) version.
